pkg/rid/store/cockroach/store.go

```diff
@@ -47,12 +47,13 @@ type Store struct {
 // Interact implements store.Interactor interface.
 func (s *Store) Interact(ctx context.Context) (repos.Repository, error) {
 	logger := logging.WithValuesFromContext(ctx, s.logger)
+	storeVersion, err := s.GetVersion(ctx)
```

Comment:
Querying the database version every time an interaction is performed seems like an optimization opportunity -- could we check the version just once upon creation of the Store?

Reply:
Never mind; we would need to do this to determine when the database was upgraded to 3.1.0.